src/coreclr/jit/codegenwasm.cpp

```diff
@@ -28,7 +24,24 @@ void CodeGen::genFnEpilog(BasicBlock* block)
     }
 #endif // DEBUG
 
-    NYI_WASM("genFnEpilog");
+    ScopedSetVariable<bool> _setGeneratingEpilog(&compiler->compGeneratingEpilog, true);
+
+#ifdef DEBUG
+    if (compiler->opts.dspCode)
+        printf("\n__epilog:\n");
+#endif // DEBUG
+
+    bool jmpEpilog = block->HasFlag(BBF_HAS_JMP);
+
+    if (jmpEpilog)
+    {
+        NYI_WASM("genFnEpilog: jmpEpilog");
+    }
+
+    // TODO-WASM: shadow stack maintenance
+    // TODO-WASM-CQ: do not emit "return" in case this is the last block
```

Comment:
Is `CQ` "code quality" in this context?

Reply:
Yes, it's a reminder that there is an opportunity to emit "better" code that we're not taking advantage of yet.